logging/taillogs/taillogs_test.go

```diff
@@ -72,21 +73,12 @@ func TestTailLogs(t *testing.T) {
 		adminClient.DeleteLog(ctx, logID)
 	}()
 
-	// following implements custom 2 min timeout for running tailLogs() sample
-	success := make(chan int, 1)
-	go func() {
+	// 5 minute timeout and 3 retries
+	testutil.Retry(t, 3, 5*time.Minute, func(r *testutil.R) {
```

Comment:
Should the total time remain 2 minutes? Increase by 3 minutes in the case the test fails because of the backend can be sensitive for running the tests.

Reply:
Changed.